Makefile

```diff
@@ -68,9 +76,12 @@ distclean: $(patsubst %, %_distclean, ${SUBMODULES}) cw_alarm_agent_distclean
 	rm -rf ${ROOT}/build
 
 include build-infra/cw-deb.mk
-
 .PHONY: deb
 deb: build deb-only
 
+include build-infra/cw-rpm.mk
+.PHONY: rpm
+rpm: build rpm-only
+
 .PHONY: all build test clean distclean
```

Comment:
I think our `.PHONY` statements here are a bit broken - `all` doesn't seem to be a target, and above we have `cdiv_handler.so`, which is a target but definitely not phony.  Would you mind taking a quick look?